

def realAddition(a,b,c):
    try:
        print("inside real Addition")
        return a+b+c
    except Exception as e:
        print(e)
        return False